C-set/1740C-Bricks-and-bags.cpp

```diff
@@ -1,46 +1,34 @@
-#pragma GCC optimize("Ofast,no-stack-protector,unroll-loops,fast-math,O3")
-#pragma GCC target("sse,sse2,sse3,ssse3,sse4,popcnt,abm,mmx,avx,avx2,fma")
- 
 #include "bits/stdc++.h"
-#include "ext/pb_ds/assoc_container.hpp"
-#include "ext/pb_ds/tree_policy.hpp"
-using namespace __gnu_pbds;
-using namespace std::chrono;
+
 using namespace std;
- 
 
- 
- 
-//------------------------------------Defines-------------------------------//
-#define ll  long long
-//#define ull unsigned long long
+#define ll long long
 #define ld long double
 
- 
 #define getline_clear cin.ignore()
-#define merge_arrays(a,b,c) merge(all(a),all(b),back_inserter(c))
-#define uid(a, b) uniform_int_distribution<int>(a, b)
+#define merge_arrays(a, b, c) merge(all(a), all(b), back_inserter(c))
+#define uid(a, b) uniform_int_distribution < int > (a, b)
 #define pb push_back
 #define eb emplace_back
 #define F first
 #define S second
- typedef long long LL;
-typedef pair<ll, ll> pii;
-typedef pair<LL, LL> pll;
-typedef pair<string, string> pss;
-typedef vector<long long int> vi;
-typedef vector<vi> vvi;
-typedef vector<pii> vii;
-typedef vector<LL> vl;
-typedef vector<vl> vvl;
-typedef map<ll,ll> mll;
+typedef long long LL;
+typedef pair < ll, ll > pii;
```

Comment:
these should be like this -> `typedef pair<ll, ll> pii`. No space required after and before the angle brackets